src/main/java/io/kestra/plugin/snmp/package-info.java

```diff
@@ -0,0 +1,8 @@
+@PluginSubGroup(
+    title = "SNMP",
+    description = "Tasks to interact with SNMP for network management and monitoring.",
+    categories = PluginSubGroup.PluginCategory.TOOL
```

Comment:
We definitely miss a lot of categories here (`MONITORING` for instance)

```suggestion
    categories = { PluginSubGroup.PluginCategory.TOOL, PluginSubGroup.PluginCategory.ALERTING }
```